torch/fx/passes/utils/matcher_utils.py

```diff
@@ -53,7 +53,8 @@ def __init__(self, pattern: Graph,
                  match_output: bool = False,
                  match_placeholder: bool = False,
                  remove_overlapping_matches: bool = True,
-                 ignore_literals: bool = False) -> None:
+                 ignore_literals: bool = False,
```

Comment:
btw with ignore literals change we should really fix various delegate code in executorch